integration-tests/features/test_create_rules_and_bullseye_import.feature

```diff
@@ -0,0 +1,65 @@
+Feature: GoogleTestWithBullseyeAndVsProject
+
+  This test verifies that analysis is able to import bullseye coverage reports and import custom rules reports
```

Comment:
Could you add a more detailed description what the test is doing please. Makes it easier for others to understand and extend it. THX.